exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,16 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number >= 0)
+  {
+    return "positive";
+  }
+  else
+  {
+    return "negative";
+  }
+}
 
```

Comment:
Perfect. I think you are getting the hang of functions that contain loops inside them. Good job